odoo/addons/base/models/res_lang.py

```diff
@@ -157,18 +166,18 @@ def fix_datetime_format(format):
     def install_lang(self):
         """
 
-        This method is called from odoo/addons/base/base_data.xml to load
+        This method is called from odoo/addons/data/res_lang_data.xml to load
         some language and set it as the default for every partners. The
-        language is set via tools.config by the RPC 'create' method on the
+        language is set via tools.config by the '_initialize_db' method on the
         'db' object. This is a fragile solution and something else should be
         found.
 
         """
         # config['load_language'] is a comma-separated list or None
         lang_code = (tools.config.get('load_language') or 'en_US').split(',')[0]
-        lang = self._lang_get(lang_code)
+        lang = self._activate_lang(lang_code)
```

Comment:
Well, your API suggests a very nice idiom here:
```suggestion
        lang = self._activate_lang(lang_code) or self._create_lang(lang_code)
```